src/plugins/dashboard/public/application/listing/dashboard_listing.tsx

```diff
@@ -43,6 +43,30 @@ import { DASHBOARD_PANELS_UNSAVED_ID } from '../../services/dashboard_session_st
 const SAVED_OBJECTS_LIMIT_SETTING = 'savedObjects:listingLimit';
 const SAVED_OBJECTS_PER_PAGE_SETTING = 'savedObjects:perPage';
 
+interface DashboardSavedObjectUserContent extends UserContentCommonSchema {
+  attributes: {
+    title: string;
+    description?: string;
+    timeRestore: boolean;
+  };
+}
+
+const toTableListViewSavedObject = (
+  savedObject: Record<string, unknown>
+): DashboardSavedObjectUserContent => {
+  return {
+    id: savedObject.id as string,
+    updatedAt: savedObject.updatedAt! as string,
+    references: savedObject.references as SavedObjectReference[],
+    type: 'dashboard',
+    attributes: {
+      title: (savedObject.title as string) ?? '',
+      description: savedObject.description as string,
```

Comment:
why is `as` necessary here?

Reply:
Because `savedObject` is not typed and declared as `Record<string, unknown>` and I need it typed to comply with `DashboardSavedObjectUserContent`.